client/src/views/auth/Signin.vue

```diff
@@ -1,82 +1,40 @@
 <template>
-  <sbc-signin @sync-user-profile-ready="onReady()" />
+  <sbc-signin
+    :idp-hint="idpHint"
+    :redirect-url-login-fail="redirectUrlLoginFail"
+    @sync-user-profile-ready="onReady()"
+  />
 </template>
 
 <script lang="ts">
 // Libraries
-import Axios from 'axios'
-import { Component, Vue } from 'vue-property-decorator'
-import { addAxiosInterceptors } from 'sbc-common-components/src/util/interceptors'
-import newReqModule from '@/store/new-request-module'
-
-// Services
-import AuthServices from '@/services/auth.services'
-import BusinessServices from '@/services/business.services'
+import { Component, Vue, Prop } from 'vue-property-decorator'
 
 // Components
 // @ts-ignore
 import SbcSignin from 'sbc-common-components/src/components/SbcSignin'
 
-// Models
-import { BusinessRequest, CreateNRAffiliationRequestBody } from '@/models/business'
-
-const axios = addAxiosInterceptors(Axios.create())
-
 @Component({
   components: {
     SbcSignin
   }
 })
 export default class Signin extends Vue {
-  /** get nr data from Session */
-  get nr () {
-    return JSON.parse(sessionStorage.getItem('NR_DATA'))
-  }
+  @Prop({ default: 'bcsc' }) idpHint: string
+  @Prop({ default: '' }) redirectUrl: string
+  @Prop({ default: '' }) redirectUrlLoginFail: string
 
   /** called when keycloak session is ready. */
   private async onReady () {
-    const currentOrganizationId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT')).id
-    const requestBody: CreateNRAffiliationRequestBody = {
-      businessIdentifier: this.nr?.nrNum,
-      phone: this.nr?.applicants?.phoneNumber || '',
-      email: this.nr?.applicants?.emailAddress || ''
-    }
-
-    try {
-      // Request to affiliate NR to current account
-      const nrResponse = await AuthServices.createNRAffiliation(currentOrganizationId, requestBody)
-      if (nrResponse?.status === 201) {
-        // update the legal api if the status is success
-        const filingBody: BusinessRequest = {
-          filing: {
-            header: {
-              name: 'incorporationApplication',
-              accountId: currentOrganizationId
-            },
-            business: {
-              legalType: this.nr?.entity_type_cd
-            },
-            incorporationApplication: {
-              nameRequest: {
-                legalType: this.nr?.entity_type_cd,
-                nrNumber: this.nr?.nrNum
-              }
-            }
-          }
-        }
-        const filingResponse = await BusinessServices.createBusiness(filingBody)
```

Comment:
Did this code move somewhere else?

Reply:
No, it became obsolete. In ticket #5332, this will be redone, in a fashion.